server/index.js

```diff
@@ -23,6 +23,39 @@ app.use('/api', (req, res, next) => {
   next(new ClientError(`cannot ${req.method} ${req.originalUrl}`, 404));
 });
 
+app.get('/api/users', (req, res, next) => {
+  const users = `
+    select "userName",
+          "imageUrl",
+          "location",
+          "dogName"
+    from "users"
```

Comment:
`select "userName",
 "imageUrl",
"location",
"dogName"
from "users"
where "location" = 'Los Angeles' and
"userId" != 6 
`